### PR TITLE
Edited the colour codes page to add some automatically highlighted colours.

### PR DESCRIPTION
**Pull request by dayofni**
### Problem

Wanted to add some of the in-game uses of colour. Fixed some issues with the formatting.

### Context

The page wasn't quite complete yet, so I wanted to add the last bits of information.
I'm not 100% sure if all of the colours are correct, but hopefully, they're close enough.

### CODE REVIEW

**Comment by seanmakesgames** [comment]
Tried to do the rebase for you on this one, but could not because don't have push perms to your repo.

Would love to get these changes in if we can. :)

File: docs/scripting/syntax/colors.mdx
Comment (matr1x-hackmud) [inline docs/scripting/syntax/colors.mdx:2]:
```suggestion
title: Colors
```

File: docs/scripting/syntax/colors.mdx
Comment (matr1x-hackmud) [inline docs/scripting/syntax/colors.mdx:22]:
```suggestion
| 0                       | ((%0#9B9B9B%))  |
```

File: docs/scripting/syntax/colors.mdx
Comment (matr1x-hackmud) [inline docs/scripting/syntax/colors.mdx:108]:
```suggestion
| ((%Sstandard text%))                                                                                                             | The default string color. Uses color ((%SS%)).                                                                                                                                                                                                                                   |
| ((user.script))                                                                                                                  | Script name highlighting. The script author uses color ((%CC%)). The script name uses color ((%LL%)).                                                                                                                                                                            |
| ((trust.script))                                                                                                                 | See above, but for Trust scripts. The user name for Trust scripts are highlighted with color ((%FF%)).                                                                                                                                                                           |
| ((9Q7T199B254M740K991GC))                                                                                                        | GC string highlighting. The numbers use color ((%BB%)), the "((%CGC%))" at the end uses color ((%CC%)). The following colors match to the following denominations: ((%NN%)) -> ((1KGC)), ((%LL%)) -> ((1MGC)), ((%JJ%)) -> ((1BGC)), ((%TT%)) -> ((1TGC)), ((%DD%)) -> ((1QGC)). |
| ((%IVOID_LAMBDA_2%))                                                                                                             | Sectors are automatically highlighted with the sector's axiom color (detailed below).                                                                                                                                                                                            |
| ((%qDATA%))                                                                                                                      | The axiom DATA uses the color ((%qq%)).                                                                                                                                                                                                                                          |
| ((%NKIN%))                                                                                                                       | The axiom KIN uses the color ((%NN%)).                                                                                                                                                                                                                                           |
| ((%lFORM%))                                                                                                                      | The axiom FORM uses the color ((%ll%)).                                                                                                                                                                                                                                          |
| ((%IVOID%))                                                                                                                      | The axiom VOID uses the color ((%II%)).                                                                                                                                                                                                                                          |
| ((%DCHAOS%))                                                                                                                     | The axiom CHAOS uses the color ((%DD%)).                                                                                                                                                                                                                                         |
| ((%FCHOICE%))                                                                                                                    | The axiom CHOICE uses the color ((%FF%)).                                                                                                                                                                                                                                        |
| ((%nLAW%))                                                                                                                       | The axiom LAW uses the color ((%nn%)).                                                                                                                                                                                                                                           |
| ((%CSPC_9999%))                                                                                                                  | "Gray sectors" use the color ((%CC%))                                                                                                                                                                                                                                            |
| ((%EWILD%))                                                                                                                      | The axiom WILD uses the color ((%EE%)).                                                                                                                                                                                                                                          |
| ((key:"value"))                                                                                                                  | Object keys use the color ((%NN%)), values use the color ((%VV%)).                                                                                                                                                                                                               |
| ((%D:::TRUST COMMUNICATION:::%))                                                                                                 | Trust communications use the color ((%DD%)).                                                                                                                                                                                                                                     |
| ((%C@%))((%Juser%)), ((%C@%))((%Kuser%)), ((%C@%))((%Muser%)), ((%C@%))((%Wuser%)), ((%C@%))((%Luser%)), and ((%C@%))((%Buser%)) | Usernames can use six different colors. The ((%C@%)) uses color ((%CC%)), and the name itself can take on any of the colors JKMWLB.                                                                                                                                              |
| ((%A3019%))((%BAD%)) ((%CD%))((%L123%))                                                                                          | This date format has four elements. The year is highlighted with color ((%AA%)), the day with color ((%LL%)). "((%BAD%))" is colored with color ((%BB%)), and "((%CD%))" is colored with color ((%CC%)).                                                                         |

```

Comment (matr1x-hackmud) [reply docs/scripting/syntax/colors.mdx:108]: Big suggest here, but we can leverage our color code plugin's automatic highlighting of client-highlighted strings here. Also, a lot of the axiom colors needed correcting

**Review by matr1x-hackmud — CHANGES_REQUESTED** [review]
Thank you for adding these! Spotted a few things that I'd want corrected before we merge

**Comment by Fayti1703** [comment]
> Tried to do the rebase for you on this one, but could not because don't have push perms to your repo.

You should be able to push regardless -- this PR has the "allow edit from maintainers" flag set, so _as long as this PR is open_ you can push to the source branch (in theory, anyway).

I'll give it a try later; see if it works on my end.

**Comment by seanmakesgames** [comment]
> > Tried to do the rebase for you on this one, but could not because don't have push perms to your repo.
> 
> You should be able to push regardless -- this PR has the "allow edit from maintainers" flag set, so _as long as this PR is open_ you can push to the source branch (in theory, anyway).
> 
> I'll give it a try later; see if it works on my end.

please do. It did not work for me.

**Comment by Fayti1703** [comment]
> > I'll give it a try later; see if it works on my end.
> 
> please do. It did not work for me.

Worked on my end -- curious if you've maybe set up the remote incorrectly? Might be worth looking into at some point.

**Comment by seanmakesgames** [comment]
> Worked on my end -- curious if you've maybe set up the remote incorrectly? Might be worth looking into at some point.

glad you tried. likely this is what was wrong when I tried.

**Review by dayofni — COMMENTED** [review]
looks good

**Review by matr1x-hackmud — APPROVED** [review]

**Pull request merged**
